packages/web-api/src/methods.ts

```diff
@@ -1884,7 +1884,7 @@ interface FileUpload {
 
 export interface FilesUploadV2Arguments extends FileUploadV2, WebAPICallOptions, TokenOverridable {
   file_uploads?: Omit<FileUploadV2, 'channel_id' | 'channels' | 'initial_comment' | 'thread_ts'>[];
-  request_file_info?: boolean;
+  request_file_info?: boolean; // since v7, this flag is no longer used
```

Comment:
Maybe we can add a JSdoc description to this parameter with the `@deprecated` tag, to make clear not to use this?

Reply:
Alternatively, since we plan on releasing this in a major version, we can drop this argument altogether.

Reply:
Thanks! Removing it may prevent smooth upgrade for many people, so I've added a deprecation comment for it.